Makefile

```diff
@@ -24,7 +24,7 @@ TOKEN ?=
 KUBECTL ?= kubectl
 SCHEDULE ?= 0 5 * * *
 INTERVAL ?= 86400
-MEMORY_REQUESTS ?= 200Mi
+MEMORY_REQUESTS ?= 1.6Gi
```

Comment:
I am against this default. It breaks other consumers. This variable is there in order to be customized. Feel free to do so from outside.